src/pages/AdminManageDul.js

```diff
@@ -150,15 +187,15 @@ class AdminManageDul extends Component {
     this.setState({
       createWarning: (election.status === 'Open'),
       showDialogCancel: true,
-      payload: election,
+      payload: election
     });
   };
 
-  openDialogCreate = (status, archived) => (e) => {
+  openDialogCreate = (election) => (e) => {
     this.setState({
-      createWarning: (status === 'Closed' && archived !== true),
+      createWarning: (election.status === 'Closed' && election.archived !== true),
```

Comment:
Instead of explicitly comparing against a boolean I'd make this `... && !election.archived`